region/wreckedship/main/Bowling Alley.json

```diff
@@ -1506,6 +1529,52 @@
         "Screw Attack is still needed to clear the bomb blocks."
       ]
     },
+    {
+      "link": [3, 4],
+      "name": "Robot Kago, R-Mode Pause Abuse Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "f_DefeatedPhantoon",
+        {"or": [
+          "h_bombThings",
+          {"and": [
+            "ScrewAttack",
+            "Morph",
+            {"or": [
+              "Plasma",
+              {"and": [
+                {"notable": "Missile Chozo Camera Unlock"},
+                "canCameraManip",
+                "canHeroShot"
+              ]}
+            ]}
+          ]}
+        ]},
+        "h_CrystalFlashForReserveEnergy",
+        "h_shinechargeMaxRunway",
+        "canKago",
+        {"autoReserveTrigger": {}},
+        "canRModePauseAbuseSparkInterrupt",
+        {"or": [
+          "canInsaneJump",
```

Comment:
I'd put off camera movement for the blind movement bits.
Insane jump vs trickyjump doesn't mean much at this level.

Reply:
There's no blind movement on the robot area strats. Even if PBs are needed to get into the Robot area, the CF can be saved until the Chozo statue and then that's the last PB you need until you end R-Mode.

Oh I guess there's the brief moment due to auto-reserve.

Reply:
I couldn't figure out the robot strat; what are the inputs for landing on top?  All I saw was it shortened knockback so you could jump sooner.
But Having to morph and unmorph made it harder than the alternatives for me.

Reply:
If you are jumping above the robot before the shinespark interrupt, then it is possible to do the same with a robotless pit, its just really hard.  
Doing all that makes the movements really complex and I'd rather do some blind movement in the empty pit and take an extra spike hit.  You think the kago is reasonable?

Reply:
I still need to look into the empty pit method. Part of why I stuck with kago originally was in previous testing I wasn't finding a way to get clear of the pit without help from the robot's hitbox (I think something about the reduced hitbox, transiting to angle spark (which would momentarily go into an upright jumping pose) pushes Samus up, which doesn't happen without a robot).

And when I was landing back in the pit... it usually resulted in multiple and very rapid spike hits that makes doing so on 0+1 suitless very questionable.